bokehjs/src/coffee/renderer/tile/tile_renderer.coffee

```diff
@@ -24,6 +27,20 @@ class TileRendererView extends PlotWidget
     @y_range = @map_plot.get('y_range')
     @y_mapper = this.map_frame.get('y_mappers')['default']
     @extent = @get_extent()
+    @_add_attribution()
+
+  _add_attribution: () ->
+    if @mget('tile_source').get('attribution')?
+      div_check = Bokeh.$('.bk-canvas-overlays').children('#tile_attribution')
```

Comment:
You can't use `Bokeh` (a global variable) here. Usually `@$(...)` is sufficient, but in this case more work may be needed, e.g. use an API function.